label-colorizer/label_colorizer.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3.11
+
+import typer
+import pathlib
+import rasterio
+import numpy as np
+from rasterio.enums import ColorInterp
+import colorsys
+import tqdm
+
+app = typer.Typer()
+
+
+@app.command()
+def main(
+    input_path: pathlib.Path,
+    output_path: pathlib.Path,
+    mask_value: int = 0,
+    seed: int = 42,
+    randomize: bool = True,
+):
+    with rasterio.open(input_path) as in_dataset:
+        assert in_dataset.count == 1  # one band
```

Comment:
Use exceptions.

Reply:
I think assertions are fine for stating assumptions in code. I'll add a better error message if it fails though.